data/hai/hai ships.txt

```diff
@@ -973,7 +988,7 @@ ship "Sea Scorpion" "Sea Scorpion (Ionic)"
 		"Ion Cannon"
 		"Ionic Blaster Prototype" 2
 		"Ionic Turret Prototype"
-		"Pulse Rifle" 15
+		"Pulse Rifle" 36
```

Comment:
What is this about?

Reply:
oh I was originally using it but then not and I didnt cancel the change
it's about prototype ships having good defense

Reply:
Ah, OK.

Reply:
```suggestion
		"Pulse Rifle" 15
```
it shouldnt be changed here tho you're right